studio/schemas/documents/salaryAndBenefits.ts

```diff
@@ -1,27 +1,37 @@
-import { defineField, defineType } from 'sanity';
-import { pageSlug } from '../schemaTypes/slug';
-import seo from '../objects/seo';
-import { title } from '../fields/text';
-import { benefitId } from './benefit';
+import { defineField, defineType } from "sanity";
+import { pageSlug } from "../schemaTypes/slug";
+import seo from "../objects/seo";
+import { title } from "../fields/text";
+import { benefitId } from "./benefit";
 
-export const salaryAndBenefitsId = "salaryAndBenefits";
+export const salaryAndBenefitsId = {
+  _root: "salaryAndBenefits",
+  showSalaryCalculator: "showSalaryCalculator",
```

Comment:
since we don't know if this is needed just yet, maybe we can add it later? 

it is not wrong in any sense, and actually a good approach, but then I would also want to make the same adjustments to other id variables.. 

I also never investigated why using const in groq returned an error. the thought was to use them in queries, but it sort of went forgotten. 

We can look into it and then whatever solution works we can apply this to the other queries as well?

Aaaand with the new versions we could also adjust with defineQuery 🤩 .. then again a bigger adjustment maybe: 
https://www.sanity.io/changelog/53b04484-04b7-4af8-a796-2ba8a819bb00#f6f340aa0152